rock-paper-scissors.js

```diff
@@ -0,0 +1,78 @@
+function computerPlay() {
+  let computerHand = Math.floor(Math.random() * 3);
+  if (computerHand === 0) {
+    return "rock";
+
+  }
+  else if (computerHand === 1) {
+    return "paper";
+
+  }
+  else if (computerHand === 2) {
+    return "scissors";
+
+  }
+}
+
+function playRound(playerSelection, computerSelection) {
+  playerSelection = playerSelection.toLowerCase();
+  if (playerSelection === "rock") {
+    if (computerSelection === "rock") {
+      return `It's A Draw! You Both Chose ${playerSelection}`
```

Comment:
A second way of improving this would be to have a function that prints `draw` and `win` and `lose`. This prevents repeated code, allows less opportunity for bugs to occur, and makes the text easier to change in the future. Making code easy to change, especially where it requires minimal effort to refactor, is a hallmark of a great coder.